tests/test_replication_key.py

```diff
@@ -4,71 +4,69 @@
 
```

Comment:
Instead of looking at the diff for this file, recommend just manually comparing them. Diff is kinda nasty.

Basically, 
1. Pulled out the setup/teardown bits and put them in their own functions. Collapsed some of the fixtures into those since they aren't used by anything else.
2. Dropped the setup/teardown into the resource method for the taptestclass
3. Moved the core of the actual test into the taptestclass

Reply:
It's not really a problem, with the other PR's we were changing a bunch of core things. This one is pretty focused I don't find it too hard to read!